packages/manager/modules/carrier-sip/src/dashboard/index.scss

```diff
@@ -0,0 +1,11 @@
+carrier-sip-dashboard { /* stylelint-disable-line */
+  @import '~bootstrap4/scss/_functions';
+  @import '~bootstrap4/scss/_variables';
+  @import '~bootstrap4/scss/_mixins';
+  @import '~bootstrap4/scss/_utilities';
+  @import '~bootstrap4/scss/_grid';
+
+  .oui-page-header {
+    margin: 0 !important;
```

Comment:
Should be a more specific selector if `!important` is needed

Reply:
The actual one is linked to the component name (eg: `carrier-sip-dashboard .oui-page-header`).

Could you tell what you had in mind to tackle the `!important` flag? Because I'm not sure to understand which selector we can used here.

Actual selector that affect us is:

```css
:not(.widget-presentation) {
  header:not(.widget-presentation-header):not(.page-header) {
```